neon_intrinsics/advsimd.md

```diff
@@ -3109,6 +3117,7 @@ The intrinsics in this section are guarded by the macro ``__ARM_NEON``.
 | <code>poly8x8_t <a href="https://developer.arm.com/architectures/instruction-sets/intrinsics/vcreate_p8" target="_blank">vcreate_p8</a>(uint64_t a)</code>     | `a -> Xn`              | `INS Vd.D[0],Xn`      | `Vd.8B -> result` | `v7/A32/A64`              |
 | <code>poly16x4_t <a href="https://developer.arm.com/architectures/instruction-sets/intrinsics/vcreate_p16" target="_blank">vcreate_p16</a>(uint64_t a)</code>  | `a -> Xn`              | `INS Vd.D[0],Xn`      | `Vd.4H -> result` | `v7/A32/A64`              |
 | <code>float64x1_t <a href="https://developer.arm.com/architectures/instruction-sets/intrinsics/vcreate_f64" target="_blank">vcreate_f64</a>(uint64_t a)</code> | `a -> Xn`              | `INS Vd.D[0],Xn`      | `Vd.1D -> result` | `A64`                     |
+| <code>fpm8x8_t <a href="https://developer.arm.com/architectures/instruction-sets/intrinsics/vcreate_f8" target="_blank">vcreate_f8</a>(uint64_t a)</code>      | `a -> Xn`              | `INS Vd.D[0],Xn`      | `Vd.8B -> result` | `v7/A32/A64`              |
```

Comment:
Is aarch32 support planned as well for the moving/casting intrinsics? In any case, this shouldn't include v7 in the list. Same with many other intrinsics in this page

Reply:
Yes, all these are meant only for A64, I forgot to edit.

Reply:
Done.